source/documentation/requirements.html.md

```diff
@@ -0,0 +1,142 @@
+          cloud7-openstack-common-testing: python-eventlet-0.17.4-4.el7
+          cloud7-openstack-newton-release: python-eventlet-0.18.4-2.el7
+          cloud7-openstack-newton-testing: python-eventlet-0.18.4-2.el7
+          cloud7-openstack-ocata-release: python-eventlet-0.18.4-2.el7
+          cloud7-openstack-ocata-testing: python-eventlet-0.18.4-2.el7
+          cloud7-openstack-pike-testing: python-eventlet-0.20.1-2.el7
+        master-distgit: https://github.com/rdo-common/python-eventlet.git
+        review-origin: null
+        review-patches: null
+        tags:
+          dependency: null
+        maintainers: 
+        - apevec@redhat.com
+        - hguemar@fedoraproject.org
+
+    If the package is found, it's already in RDO repositories and no more actions are needed to add
+    it to the repos.
+    
+    <br />
+4. In case that the dependency is not in CentOS base or CloudSIG repo, you can check if it has been built
+by other SIGs in [CBS web interface](http://cbs.centos.org/koji/). You can use wildcards in the packages
+search expression. If you find the desired dependency, you can open a bug in [Red Hat Bugzilla for
+RDO product](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO&component=distribution) requesting
+the inclussion of the package in RDO repos. RDO Core members will handle the request.
+    
+    <br />
+5. If the new package is not in CBS, you must check if it's packaged in Fedora using the [package
+browser](https://apps.fedoraproject.org/packages/). If the package exists you can open a bug in [Red Hat Bugzilla for
+RDO product](https://bugzilla.redhat.com/enter_bug.cgi?product=RDO&component=distribution) requesting
+the rebuild of the package in RDO repos. RDO Core members will rebuild the package using latest existing version
+in fedora.
+    
+    <br />
+6. When the packages doesn't exist even in fedora you need to add the package following the [New package
+process](https://fedoraproject.org/wiki/New_package_process_for_existing_contributors). Note that a fedora
+packager needs to participate in this process. While RDO core members may maintain the new package for
+common requirements used by different projects, dependencies for specific project must be maintained in
+Fedora by the project team. Once the package is included in Fedora repos you can create a bugzilla as
+explained in step 5.
+    
+    <br />
+7. Once the package is available in the repos, you can add it to the list of *Requires* or *BuildRequires* in
+your package spec file. Note that optional dependencies not used in default or common configurations should
+not be added as Requires but installed only when needed.
+
+
+### Updating a requirement in RDO CloudSIG repositories
+
+There are some rules to follow when a requirement update is needed by a OpenStack project:
+
+* If the dependency is included in upstream requirements project, the required version must be lower or equal to
```

Comment:
ideally, it should be equal u-c

If mentioning lower, g-r >= must be mentioned along with caveat that g-r lower bounds are not tested upstream and might not be working!